### PR TITLE
Doc/rpi image name

### PR DESCRIPTION
**Pull request by patrickvacek**

### CODE REVIEW

File: docs/ota-client-guide/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ ifndef::env-github[:pageroot:]
 * xref:{pageroot}generating-provisioning-credentials.adoc[Get a provisioning Key]
 * xref:{pageroot}build-images.adoc[Use our sample recipes to build disk images]
 ** xref:{pageroot}build-raspberry.adoc[Build for a Raspberry Pi]
-** xref:{pageroot}build-quemu.adoc[Build for QEMU]
+** xref:{pageroot}build-qemu.adoc[Build for QEMU]
```
Comment (merltron) [inline docs/ota-client-guide/modules/ROOT/nav.adoc]: @patrickvacek FYI: Changing a file name will break an include in another repo (the "Get Started" repo) - let me fix that before you merge.

Comment (patrickvacek) [reply docs/ota-client-guide/modules/ROOT/nav.adoc]: Ah, sorry. Let me know when you're ready.

Comment (merltron) [reply docs/ota-client-guide/modules/ROOT/nav.adoc]: Alright done, will click "Approve" now.

Comment (patrickvacek) [reply docs/ota-client-guide/modules/ROOT/nav.adoc]: Thanks!

**Pull request merged**